weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -65,26 +76,36 @@ def __init__(
                 self._flush_calls,
                 max_queue_size=max_calls_queue_size(),
             )
-        self._auth: Optional[tuple[str, str]] = None
         self.remote_request_bytes_limit = remote_request_bytes_limit
 
+        self.stainless_client = WeaveTrace(
+            username=entity_name if entity_name else "",
```

Comment:
i don't understand why you are passing entity name around. It should always be blank

Reply:
but why is it needed? it is not used in auth